app/controllers/number_of_mentions_controller.rb

```diff
@@ -1,7 +1,13 @@
 class NumberOfMentionsController < ApplicationController
   def show
     @phrase = Phrase.find(params[:id])
-    @mentions = SurveyAnswer.find_by_sql(['select date(s.started_at) as date, count(m.id) as total_mentions from survey_phrases m join phrases p on p.id = m.phrase_id join survey_answers sa on sa.id = m.survey_answer_id join surveys s on s.id = sa.survey_id where p.id = ? group by(date(s.started_at)) limit 10;', @phrase.id])
-    @total_mentions = @mentions.inject(0){|sum, mention| sum + mention.total_mentions}
+    @mentions = mentions
+  end
+
+private
+
+  def mentions
+    SurveyPhrase.mentions_by_date_range_for_phrase(@phrase, Date.new(2020, 4, 1), Date.new(2020, 4, 7))
+      .sort_by{ |date, _| date }
```

Comment:
Formatting issues here - surprised it's not being picked up by the linter 🤔 

```suggestion
      .sort_by { |date, _| date }
```